### PR TITLE
Surface Automation

### PR DESCRIPTION
**Pull request by openbots-ff**
- Added SurfaceAutomationCommand with the ability to click, set text, set secure text, check and wait for images to exist on screen. 
- Added CaptureImageCommand to store Bitmaps as variables for use in Begin If/Begin Loop Commands.
- Added project manifest to turn on DPI Awareness to fix the previous inability to capture full screen shots.
- Fixed rendering changes made as a result of the manifest being added as well as previously existing bugs. 
- Cleaned up Image Commands.
- Added Bitmap variable type to the debug pane.

### CODE REVIEW

**Review by CameronHerwig — DISMISSED** [review]

File: taskt.Studio/Commands/Image Commands/PerformOCRCommand.cs [hunk 1]
```diff
@@ -16,26 +17,25 @@ namespace taskt.Commands
 {
     [Serializable]
     [Group("Image Commands")]
-    [Description("This command allows you to covert an image file into text for parsing.")]
-    [UsesDescription("Use this command when you want to convert an image into text.  You can then use additional commands to parse the data.")]
-    [ImplementationDescription("This command has a dependency on and implements OneNote OCR to achieve automation.")]
+    [Description("This command converts an image file into text for parsing.")]
```
Comment (ob-ab) [inline taskt.Studio/Commands/Image Commands/PerformOCRCommand.cs]:
```suggestion
    [Description("This command extracts text from an image file.")]
```

File: taskt.Studio/Commands/Image Commands/CaptureImageCommand.cs [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    public class CaptureImageCommand : ScriptCommand
+    {
+        [XmlAttribute]
+        [PropertyDescription("Capture Search Image")]
+        [InputSpecification("Use the tool to capture an image that will be located on screen during execution.")]
+        [SampleUsage("")]
+        [Remarks("Images with larger color variance will be found more quickly than those with a lot of white space. \n" +
+                 "For images that are primarily white space, tagging color to the top-left corner of the image and setting \n" +
+                 "the relative click position will produce faster results.")]
+        [PropertyUIHelper(UIAdditionalHelperType.ShowImageRecogitionHelper)]
```
Comment (ob-ab) [inline taskt.Studio/Commands/Image Commands/CaptureImageCommand.cs]: I think `ShowImageRecogitionHelper` should be renamed to `ShowImageCaptureHelper`

**Review by ob-ab — CHANGES_REQUESTED** [review]
Please make a couple of suggested changes.

File: taskt.Studio/Commands/Image Commands/CaptureImageCommand.cs (hunk 2 above)
Comment (openbots-ff) [reply taskt.Studio/Commands/Image Commands/CaptureImageCommand.cs]: done

**Review by ob-ab — DISMISSED** [review]

**Review by CameronHerwig — APPROVED** [review]

**Review by ob-ab — APPROVED** [review]

**Pull request merged**